package-lock.json

```diff
@@ -17643,7 +17779,7 @@
         },
         "htmlparser2": {
           "version": "3.3.0",
-          "resolved": "https://registry.npmjs.org/htmlparser2/-/htmlparser2-3.3.0.tgz",
+          "resolved": "http://registry.npmjs.org/htmlparser2/-/htmlparser2-3.3.0.tgz",
```

Comment:
@wylieconlon seems like there is quite a bit of noise in the `package-lock` diff, both these switches from `https` to `http`, and dependencies where `"bundled": true` is replaced  by `resolved` and `integrity` properties. I’m not expert enough in NPM to know exactly why that is going on, and would definitely accept a confident “this is kind of inevitable because of how NPM works”, but just wondering if it is maybe avoidable.